willow/plugins/pillow.py

```diff
@@ -29,6 +29,12 @@ def _PIL_Image():
     return PIL.Image
 
 
+def _PIL_ImageOps():
+    import PIL.ImageOps
+
+    return PIL.ImageOps
+
```

Comment:
I'm following the same pattern as the code above. Not sure if it is needed though, I assume the indirection is to defer loading large modules? Not sure if that applies to `PIL.ImageOps`